android/src/com/frostwire/android/gui/activities/MainActivity.java

```diff
@@ -779,11 +673,11 @@ private void updateHeader(Fragment fragment) {
             if (fragment instanceof MainFragment) {
                 View header = ((MainFragment) fragment).getHeader(this);
                 if (header != null) {
-                    setToolbarView(header, Gravity.LEFT);
+                    setToolbarView(header, Gravity.START);
```

Comment:
`START` works better according to android guidelines, for locales which write from right to left (e.g. arabic)